src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,22 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => (
+  <ul>
+    {tabs.map(tab => (
+      <li
+        key={tab.id}
+        className={tab.id === activeTabId ? 'is-active' : ''}
```

Comment:
you can use `classNames` function from `classnames` library 

```suggestion
        className={classNames({'is-active': tab.id === activeTabId })}
```